packages/web-app-admin-settings/src/views/Spaces.vue

```diff
@@ -135,7 +135,7 @@ export default defineComponent({
     const { sideBarOpen, sideBarActivePanel } = useSideBar()
 
     const loadResourcesEventToken = ref(null)
-    const updateQuotaForSpaceEventToken = ref(null)
+    const updateQuotaForSpaceEventToken = null
```

Comment:
Needs to stay `ref(null)`. Why did you change it? 🤔

Reply:
I think that refers to a comment I made in the last PR, where I said that this does not need to be a `ref` because it's not reactive. Although you need to make it a `let` and remove `.value` (and/or `unref()`) down below.

Reply:
Ah ok. Yes, simply removing the wrapping `ref()` is not sufficient. That's why I was confused.